lib/MusicBrainz/Server/Report/RecordingsSameNameDifferentArtistsSameName.pm

```diff
@@ -0,0 +1,41 @@
+package MusicBrainz::Server::Report::RecordingsSameNameDifferentArtistsSameName;
+use Moose;
+
+with 'MusicBrainz::Server::Report::RecordingReport',
+     'MusicBrainz::Server::Report::FilterForEditor::RecordingID';
+
+sub query {
+    "
+    SELECT
+        recording_id,
+        row_number() OVER (ORDER BY musicbrainz_collate(rname), recording_id)
```

Comment:
It would be much better for the report to be ordered by recording name, then artist *id*. Right now, while the MBIDs make it easy to spot which artists are probably wrong, the wrong MBIDs are not sorted next to each other at all.